backend/src/mocks/httpApi.ts

```diff
@@ -0,0 +1,26 @@
+import { HexString } from "@polkadot/util/types";
+
+import { ChainName } from "../types";
+import HttpApiDefault from '../httpApi';
```

Comment:
But there is no `HttpApiDefault` in there, how does it even compile?

Reply:
it's default export, I can import it as any name

Reply:
I feel stupid :facepalm: